application-accelerator/configuration.hbs.md

```diff
@@ -421,6 +421,80 @@ spec:
 
 If you are using the Tanzu CLI, then add the `--secret-ref` flag to your `tanzu accelerator create` command and provide the name of the secret for that flag.
 
+## <a id="enabling-tls"></a> Enabling TLS for Accelerators Server
+
+In order to enable TLS for Accelerators Server, following properties should be provided in the `accelerator` section of the `tap-values.yaml` file:
+
+```yaml
+server:
+  tls:
+    enabled: true
+    key: <SERVER-PRIVATE-KEY>
+    cert: <SERVER-CERTIFICATE>
+```
+
+Where:
+
+- `SERVER-PRIVATE-KEY` is the pem encoded server private key.
+- `SERVER-CERTIFICATE` is the pem encoded server certificate.
+
+This is an example of a part of a `tap-values.yaml` configuration:
+
+```yaml
+server:
+  tls:
+    enabled: true
+    key: |
+      -----BEGIN PRIVATE KEY-----
+      .
+      .  < private key data >
+      .
+      -----END PRIVATE KEY-----
+    crt: |
```

Comment:
expecting `cert:` instead of `crt:` here

Reply:
To avoid confusion and keep name consistent, I replaced **"cert"** with **"crt"** everywhere.